pandas/tests/dtypes/test_common.py

```diff
@@ -107,6 +147,37 @@ def test_period_dtype(self, dtype):
         assert com.pandas_dtype(dtype) == dtype
 
 
+@pytest.mark.xfail(reason="not yet implemented")
```

Comment:
This test currently fails for all parameterizations.

It sounds like the `is_*_dtype` functions [should be changed](https://github.com/pandas-dev/pandas/pull/47108#issuecomment-1136689084) so that this tests passes, but I left that out of this PR for 2 reasons:

- Modifying only `pandas_dtype` is enough to get a (hopefully) more useful error message, which was the request in #31356.

    ```
    >>> pd.Series(dtype=pd.Int64Dtype)
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
      File "/home/pandas/pandas/core/series.py", line 395, in __init__
        dtype = self._validate_dtype(dtype)
      File "/home/pandas/pandas/core/generic.py", line 436, in _validate_dtype
        dtype = pandas_dtype(dtype)
      File "/home/pandas/pandas/core/dtypes/common.py", line 1604, in pandas_dtype
        raise TypeError(msg)
    TypeError: Must pass dtype instance, not dtype class
    ```

- Want to keep this PR narrowly scoped.